src/tools/wix/CommandLineInfo.cs

```diff
@@ -0,0 +1,43 @@
+﻿//-------------------------------------------------------------------------------------------------
+// <copyright file="CommandLineInfo.cs" company="Outercurve Foundation">
+//   Copyright (c) 2004, Outercurve Foundation.
+//   This software is released under Microsoft Reciprocal License (MS-RL).
+//   The license and further copyright text can be found in the file
+//   LICENSE.TXT at the root directory of the distribution.
+// </copyright>
+// 
+// <summary>
+// Utility class for Burn ExePackage CommandLine information.
+// </summary>
+//-------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Tools.WindowsInstallerXml
+{
+    using System;
+
+    /// <summary>
+    /// Utility class for Burn CommandLine information.
+    /// </summary>
+    internal class CommandLineInfo
```

Comment:
Mission accomplished. :) Fair point but it turns out there are few *Row classes to model. I thought I had one with WixApprovedExeForElevationRow but then I found ApprovedExeForElevation. :)